pkg/runtime/types.go

```diff
@@ -80,17 +86,19 @@ type TypeMeta struct {
 //
 // +protobuf=true
 type RawExtension struct {
-	// RawJSON is the underlying serialization of this object.
-	RawJSON []byte
+	// Raw is the underlying serialization of this object.
```

Comment:
Yeah, let's remove and add a TODO: here and a follow up issue to determine
what the solution is.  I'll open a PR on the "let callers encode themselves
from serializers" side.

On Thu, Mar 17, 2016 at 4:43 PM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/runtime/types.go
> <https://github.com/kubernetes/kubernetes/pull/23112#discussion_r56574323>
> :
>
> > @@ -80,17 +86,19 @@ type TypeMeta struct {
> >  //
> >  // +protobuf=true
> >  type RawExtension struct {
> > -	// RawJSON is the underlying serialization of this object.
> > -	RawJSON []byte
> > +	// Raw is the underlying serialization of this object.
>
> Thanks for detailed explanation - that helps a lot!
>
> So I guess, we should remove ContentEncoding and ContentType from here,
> right?
> Will do that tomorrow.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23112/files/edfb4acc188fbb26529993e02d82fd72faa247d7#r56574323>
>